packages/search/src/features/registration/deduplicate/elasticSearchTestContainer.test.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import { startContainer, stopContainer } from './elasticSearchTestContainer'
+// @ts-ignore
+import { IBirthCompositionBody } from '@search/elasticsearch/utils'
+// @ts-ignore
+import { indexComposition } from '@search/elasticsearch/dbhelper'
+import * as elasticsearch from '@elastic/elasticsearch'
+import { searchForDuplicates } from './service'
+import { StartedElasticsearchContainer } from 'testcontainers'
+
+jest.setTimeout(10 * 60 * 1000)
+
+let container: StartedElasticsearchContainer
+let client: elasticsearch.Client
+
+beforeAll(async () => {
+  container = await startContainer()
+  const host = container?.getHost() ?? '0.0.0.0'
+  const port = container?.getMappedPort(9200) ?? 9200
```

Comment:
Do these actually need defaults? I'm thinking if those commands fail, then we probably don't want this client to connect to any other elasticsearch either, right?